### ISSUE TITLE
Add option to prevent pub global run intl_utils:generate from running automatically

### ISSUE THREAD

**Issue by simonpham** [localizely/intl_utils#9]
Imagine if you have 100+ .arb files, after use **Extract to ARB** option to extract a string to all the arb files.
The plugin then runs `pub global run intl_utils:generate` 100+ times.
It should run only once. Or we should have an option to prevent it from running automatically.

**Comment by aleksakrstic** [comment]
Hi @simonpham 
It should run `pub global run intl_utils:generate` once, but that command would generate it for all languages (arb files).

Why do you think it runs 100+ times? 
Do you have performance issues, and what IDE do you use, VSCode or Android Studio?

**Comment by simonpham** [comment]
> It should run pub global run intl_utils:generate once, but that command would generate it for all languages (arb files).
Yes. This is true.
That command will run once if I made a change to an ARB file & save.
But if I made changes to multiple ARB files & save, that command will run multiple times. This is not necessary and causes a huge impact on performance.

I use IntelliJ/Android Studio.

**Comment by aleksakrstic** [comment]
Nice catch @simonpham, thanks!
I could reproduce it when using "Save All".

I am closing this issue, and I opened a new issue for Android Studio IDE plugin https://github.com/localizely/flutter-intl-intellij/issues/48 since it is related to IDE plugin, not to `intl_utils` package.
Feel free to add more info there in case I missed something.
